PullRequestClosed/PullRequestClosed.test.ts

```diff
@@ -101,13 +101,12 @@ describe('PullRequestClosed', () => {
   });
 
   it('Jira issue does not exist', async () => {
-    const logSpy = jest.spyOn(core, 'setFailed').mockImplementation(() => { });
-    try {
-      await runAction('FAKE-1234 Title');
-      expect(logSpy).toHaveBeenCalledWith('Scaffolding did not expect FAKE-1234');
-    } finally {
-      logSpy.mockRestore();
-    }
+    await runAction('FAKE-1234 Title');
+    expect(logTester.logsParams).toStrictEqual([
+      "Loading PR #42",
+      "Skipping issue cancellation for creator undefined",
```

Comment:
Please add `|| null` in `PullRequestClosed` line 31 to have `creator null` here. I was fixing these surprises when I discovered them

Reply:
Illegal